### ISSUE TITLE
Art/Buildings: two screenshots

### ISSUE THREAD

**Issue by KaylebJay** [minetest/blog#65]
![](https://imgur.com/PvEjTuh.png)
Central Ercavica, a medieval city built by doggo on Tunnelers' Abyss.
![](https://imgur.com/0FdBePa.png)
An industrial facility in Walton's Forest, a 19th-century themed city built by TomBuilder on Tunnelers' Abyss.
Screenshot by pups.

Please rephrase this as necessary, it isn't particularly clever at the moment :-)

**Comment by MisterE123** [comment]
Added. Thank you!

**Comment by rubenwardy** [comment]
Btw, the perspective on these screenshots is super weird. Looks like they were taken up close with a high FOV. I suggest taking photos from further away rather than increasing FOV. You can reduce FOV to zoom in

**Comment by KaylebJay** [comment]
I'll try to see what I can do for next time: it seems a lot of players on our server (including myself) use high FOVs like this. :)
